process_request/tests.py

```diff
@@ -55,6 +67,39 @@ def test_machineuser(self):
         self.assertEqual(str(user), system)
 
 
+class TestHelpers(TestCase):
+
+    def test_get_container_indicators(self):
+        letters = ''.join(random.sample(string.ascii_lowercase, random.randint(2, 10)))
```

Comment:
You call this particular combination of things a number of times, so I'd suggest creating a `random_string()` helper.